app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
```

Comment:
Please put every single argument to a new line
```suggestion
    def __init__(
        self, 
        nickname: str,
        favourite_dish: str,
        skill_level: int
    ) -> None:
```